local-cli/generator-macos/templates/macos/HelloWorld-macOS/Info.plist

```diff
@@ -15,29 +20,20 @@
 	<key>CFBundleName</key>
 	<string>$(PRODUCT_NAME)</string>
 	<key>CFBundlePackageType</key>
-	<string>APPL</string>
+	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
 	<key>CFBundleShortVersionString</key>
 	<string>1.0</string>
 	<key>CFBundleVersion</key>
 	<string>1</string>
 	<key>LSMinimumSystemVersion</key>
 	<string>$(MACOSX_DEPLOYMENT_TARGET)</string>
-	<key>NSAppTransportSecurity</key>
-	<dict>
-		<key>NSAllowsArbitraryLoads</key>
-		<true/>
-		<key>NSExceptionDomains</key>
-		<dict>
-			<key>localhost</key>
-			<dict>
-				<key>NSExceptionAllowsInsecureHTTPLoads</key>
-				<true/>
-			</dict>
-		</dict>
-	</dict>
```

Comment:
@tom-un I think we only need `NSAllowsLocalNetworking`, no?

Reply:
I don't know.  When I test it locally it works.  But now the mac Info.plist and iOS Info.plist are different.   Should they both be the same in terms of these network keys?   The core iOS template Info.plist is this way too.

Reply:
I’ll revert this for now and follow-up in the future.